cmd/snapctl/commands.go

```diff
@@ -117,6 +117,21 @@ var (
 						flRunning,
 					},
 				},
+				{
+					Name: "config",
```

Comment:
I'm not sure I see how showing anything else would make sense. It's displaying the output from `/v1/plugins/:type/:name/:version/config` rest call. Are there other config values that would make sense to show for a plugin?